articles/contributing-style-guide.md

```diff
@@ -181,25 +180,24 @@ We suggest:
-While we must maintain the history and intellectual provenance of concepts in quantum computing, demanding that all users be versed in this history to use even the most basic of functions and operations places a barrier to entry that is in most cases severe enough to even present an ethical compromise. -->
-Thus, we recommend that wherever reasonable, common nouns that describe a concept be adopted in strong preference to proper nouns that describe the publication history of a concept.
-As a particular example, the singly controlled SWAP and doubly controlled NOT operations are often called the "Fredkin" and "Toffoli" operations in academic literature, but are identified in Q# primarily as `CSWAP` and `CCNOT`.
-In both cases, the API documentation comments provide synonymous names based on proper nouns, along with all appropriate citations.
-
-This preference is especially important given that some usage of proper nouns will always be necessary — Q# follows the tradition set by many classical languages, for instance, and refers to `Bool` types in reference to Boolean logic, which is in turn named in honor of George Boole.
+In physics it is common to name things after the first person that published information about it. HOever, most non-physicists aren’t familiar with the important names and history of physics.
+Don't rely too heavily on naming conventions from physics since they can create barriers to entry. They require users from other backgrounds to learn a large number of names just to use common operations and concepts.
+<!-- Reduce confusion by making code more accessible.
+Quantum computing is rich with domain expertise. We must be conscienscious of the demands we place on that expertise as we design quantum software.
+Be aware of conventions from physics of the names of algorithms and operations being the names of their original publishers.
+We must maintain the history and intellectual provenance of quantum computing concepts, but to demand that all users learn the history of physics creates barriers to entry that are severe enough to present an ethical compromise. -->
+We recommend that wherever reasonable, common nouns that describe a concept be adopted. This is preferrable over proper nouns that describe the publication history of a concept.
+As an example, the singly controlled SWAP and doubly controlled NOT operations are often called the "Fredkin" and "Toffoli" operations in academic literature, but are identified in Q# primarily as `CSWAP` and `CCNOT`.
+In both cases, API documentation comments provide synonymous names based on proper nouns, along with all appropriate citations.
+
+This option is especially important since some use of proper nouns will always be necessary — Q# follows the tradition set by many classical languages, for instance, and refers to `Bool` types in reference to Boolean logic, which is in turn named in honor of George Boole.
```

Comment:
"Option" is weaker than "preference," making it sound like this paragraph can be safely ignored.

Reply:
@cgranade - Both of them imply a choice.  Is this a preference or a definite instruction?

Reply:
Agreed, both represent choices, but "option" makes it sound like a completely neutral choice, undermining the above guidance. It is a preference that authors can override based on their best judgment, but that they should follow by default.